errors/errors.go

```diff
@@ -0,0 +1,173 @@
+package errors
```

Comment:
Why create a new one when we already have it in wit?

Reply:
because I need some type of errors that are specific to tenant and openshift.

Reply:
yes, I've been thinking about it for a few packages, including `errors` and `log` amongst others. It would probably make sense to have something like `fabric8-commons` for all those shared pkgs. But that should be part of another PR, IMO